test_rendering/spec/ol/style/linestring.test.js

```diff
@@ -91,6 +91,23 @@ describe('ol.rendering.style.LineString', function() {
         })
       }));
       vectorSource.addFeature(feature);
+
+      feature = new ol.Feature({
```

Comment:
I was going to say that this new feature should be tested in a new set of tests, to avoid unnoticed rendering regressions with existing features. But then I saw that this whole set of tests is written in a way that all style options are shown in a single reference image. So it's not your fault that things are as they are, and your changes here are ok.